back/src/main/resources/templates/post.html

```diff
@@ -0,0 +1,45 @@
+{{#posts}}
```

Comment:
would be better to have list in the page itself in case we use post for post page

Reply:
Blud this is outdated. I have it in later commits